frontend/src/queries/nodes/PersonsNode/PersonsSearch.tsx

```diff
@@ -4,38 +4,57 @@ import { Tooltip } from 'lib/lemon-ui/Tooltip'
 
 import { useDebouncedQuery } from '~/queries/hooks/useDebouncedQuery'
 import { PersonsNode, PersonsQuery } from '~/queries/schema'
+import { isInsightPersonsQuery, isPersonsQuery, isRetentionQuery } from '~/queries/utils'
 
 interface PersonSearchProps {
     query: PersonsNode | PersonsQuery
     setQuery?: (query: PersonsNode | PersonsQuery) => void
 }
 
+type actorType = 'person' | 'group'
```

Comment:
```suggestion
type ActorType = 'person' | 'group'
```

The convention so far has been to capitalize types.

Reply:
Typo, will fix.